store/sql_upgrade.go

```diff
@@ -15,6 +15,7 @@ import (
 )
 
 const (
+	VERSION_3_6_0 = "3.6.0"
```

Comment:
I can submit it in a separate PR if we want to start doing that, but we [haven't done that in the past](https://github.com/mattermost/platform/commit/1e7985a87a72bea9a308cf1506dacc828c6e2e1c#diff-a788391365ece46cdf5c98badcdc3c32R18)